CHANGELOG.md

```diff
@@ -8,15 +8,13 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [1.16.0] - 2016-09-22
 ### Added
-- New rule [`no-dynamic-require`]: restrict deep package imports to specific folders. ([#567])
```

Comment:
Haha :smile:
Honestly, I would write so many more rules if I didn't have to write docs. That's pretty much a pain/time waster everytime (as I'm more fluent in code than docs). If only I had a scribe that wrote the docs for me...)
In https://github.com/jfmengels/eslint-plugin-fp/, I at least automatised updating the index.js and the README, so that solves part of my problems ^^